### ISSUE TITLE
pipeline.queue.events is updated only after logstash restart

### ISSUE THREAD

**Issue by coolprag** [elastic/logstash#8478]
Link to salesforce issue - https://elastic.my.salesforce.com/5006100000ENGKp

When getting statistics from the Logstash API - /_node/stats/pipeline - the persistent queue event count is not updated until  Logstash is restarted.

Logstash version - 5.6

There is another issue opened related to this one : https://github.com/elastic/logstash/issues/8382#issuecomment-335607722

**Comment by jakelandis** [comment]
Closing as duplicate to #8382 (abit of mis-communication on how the issues are tagged w/r/t to releases)
